Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -60,61 +133,43 @@ worker::Map<worker::EntityId, worker::Entity> CreateLevelEntities(UWorld* World)
 	{
 		for (int y = -PlaceholderCountAxis / 2; y < PlaceholderCountAxis / 2; y++)
 		{
-			const Coordinates PlaceholderPosition{x * CHUNK_SIZE + CHUNK_SIZE * 0.5f, 0, y * CHUNK_SIZE + CHUNK_SIZE * 0.5f};
-			LevelEntities.emplace(PlaceholderEntityIdCounter, improbable::unreal::FEntityBuilder::Begin()
-				.AddPositionComponent(Position::Data{PlaceholderPosition}, UnrealWorkerWritePermission)
-				.AddMetadataComponent(Metadata::Data("Placeholder"))
-				.SetPersistence(true)
-				.SetReadAcl(AnyWorkerReadPermission)
-				.AddComponent<unreal::UnrealLevelPlaceholder>(unreal::UnrealLevelPlaceholder::Data{}, UnrealWorkerWritePermission)
-				.Build());
+			const Coordinates PlaceholderPosition{ x * CHUNK_SIZE + CHUNK_SIZE * 0.5f, 0, y * CHUNK_SIZE + CHUNK_SIZE * 0.5f };
```

Comment:
This can all go when @mironec stably named snapshot changes go in.

Reply:
This potentially needs to still be here because of a bug @improbable-valentyn raised. If he's ok with removing it then we can do so.

Reply:
The problem is runtime not reassigning authority fast enough when reconnecting to a deployment with new workers.
It first assigns authority to the dead worker (regardless of how long you wait before connecting a new one), then fails all commands for the next 5 seconds, then reassigns the authority.